common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -14,14 +22,16 @@
     url = staticfiles_storage.url(file)
 except:
     url = file
-%>${url}${"?raw" if raw else ""}</%def>
+## HTML-escaping must be handled by caller
```

Comment:
I'm not sure which URL you are referring to at the end of this PR.

Here is an example of this url() call in use, and you can see that it would be escaped by virtue of the page being safe and it being used in an expression that is escaped:
https://github.com/edx/edx-platform/blob/master/lms/templates/dashboard.html#L38